scripts/gff_metaparser/genmetaconf/metaconf.py

```diff
@@ -94,6 +94,16 @@ def update(self, key, val, tech = False):
     else:
       out[key].append(val)
 
+  def normalise_asm_name(self, asm_name):
+    if not asm_name:
+      return asm_name
```

Comment:
If `asm_name` can be `None`, then this is fine, but if not, you can just run the code below (will not do much and the performance impact will be minimal).

Reply:
as expected